src/RpcServer/RpcServer.Node.cs

```diff
@@ -52,15 +52,55 @@ protected virtual JToken GetPeers(JArray _params)
 
         private static JObject GetRelayResult(VerifyResult reason, UInt256 hash)
         {
-            if (reason == VerifyResult.Succeed)
-            {
-                var ret = new JObject();
-                ret["hash"] = hash.ToString();
-                return ret;
-            }
-            else
+
+            switch (reason)
             {
-                throw new RpcException(-500, reason.ToString());
+                case VerifyResult.Succeed:
+                    {
+                        var ret = new JObject();
+                        ret["hash"] = hash.ToString();
+                        return ret;
+                    }
+                case VerifyResult.AlreadyExists:
+                    {
+                        throw new RpcException(RpcError.AlreadyExists.WithData(reason.ToString()));
+                    }
+                case VerifyResult.OutOfMemory:
+                    {
+                        throw new RpcException(RpcError.MempoolCapReached.WithData(reason.ToString()));
+                    }
+                case VerifyResult.InvalidScript:
+                    {
+                        throw new RpcException(RpcError.InvalidScript.WithData(reason.ToString()));
+                    }
+                case VerifyResult.InvalidAttribute:
+                    {
+                        throw new RpcException(RpcError.InvalidAttribute.WithData(reason.ToString()));
```

Comment:
Good! @superboyiii, that's probably the most frequently used code from NeoFS clients, could you, please, test that error codes returned by `GetRelayResult` match the NeoGo ones (and specification)? https://github.com/nspcc-dev/neo-go/blob/d90169761595933febb8a25d1abde2e46154b96d/pkg/services/rpcsrv/server.go#L2636